test/migration/UpgradeFromSafe120.spec.ts

```diff
@@ -37,10 +37,10 @@ describe("Upgrade from Safe 1.2.0", () => {
 
         expect(await safe.VERSION()).to.be.eq("1.2.0");
         const nonce = await safe.nonce();
-        const data = ChangeMasterCopyInterface.encodeFunctionData("changeMasterCopy", [singleton140]);
+        const data = ChangeMasterCopyInterface.encodeFunctionData("changeMasterCopy", [singleton150]);
```

Comment:
In case its missing, can we add an `UpgradeFromSafe141` test spec too?

Reply:
Added both `UpgradeFromSafe141` and `UpgradeFromSafe130`.

Reply:
Added 👍🏾